src/main/java/com/example/bookstore/exception/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package com.example.bookstore.exception;
+
+public class RegistrationException extends RuntimeException {
```

Comment:
imo need to extends Exception because you throw it in method signature

Reply:
agree, make it checked exception, or do not declare in method signature at all